aesara/compile/builders.py

```diff
@@ -3,15 +3,20 @@
 from collections import OrderedDict
 from functools import partial, reduce
 
-import aesara
 from aesara import tensor as aet
```

Comment:
I think this import still goes through the main `__init__.py`? (cc @brandonwillard )

Reply:
Are you referring to `aesara/__init__.py`?

Reply:
Yes. My understanding is that `from aesara import something` goes through the `aesara/__init__.py` even if `something` is a submodule?

Reply:
It should, and so should every `import` under `aesara` (e.g. the `from aesara import tensor` line).  Those `ii

Reply:
Thank's for the distinction between (non-)user-level APIs.

Without even analyzing what kind of API it's about, we can replace imports like
```
from aesara import tensor as aet
```
with
```
import aesara.tensor as aet 
```
everywhere in the codebase. (cc @CloudChaoszero )

Reply:
Those are supposed to be equivalent statements in Python, but, due to a bug in Python 3.6, I believe the latter form will give rise to cyclic import issues that shouldn't actually be a problem.

Reply:
Thanks for the feedback! I updated all 22 file changes to ensure there is no related `from aesara import....`